settings.py

```diff
@@ -1193,8 +1193,6 @@ def _pkce_required(client_id):
 
 ALLOW_MAKE_SUPERUSER_COMMAND = True
 
-ALLOW_SUPERUSER_MANAGEMENT = True
-
```

Comment:
It seems to me the code change doesn't match the commit message

> Remove ALLOW_MAKE_SUPERUSER_COMMAND variable in favor of using just t…
> …he ALLOW_SUPERUSER_MANAGEMENT variable

I think the code is correct and the commit message was just a simple mistake? If that's right, then I think it's worth amending the commit message.